app/models/steps/approval.rb

```diff
@@ -0,0 +1,4 @@
+module Steps
```

Comment:
was wondering if this should be plural or singular...seems like plural is how Rails does it (eg: http://api.rubyonrails.org/classes/ActiveRecord.html)...good lesson for me, I default to singular! so :+1: on this naming